README.md

```diff
@@ -61,7 +64,7 @@ You should see the following modules:
 ```
 Now you can finally build the repository by running: 
 ```bash
-mvn install
+mvn clean install
```

Comment:
Not sure the clean phase is required as this is likely to be a fresh clone of the code

Reply:
It is not necessary, but I think that it is a good practice.  It will set the user-up the command prompt with the inevitable variant when there is a need to re-do the build

Reply:
I dont mind having the clean in here and it is good practice